src/api/components/modelRoute/modelRoute.service.js

```diff
@@ -0,0 +1,77 @@
+
+const ModelRouteModel = require('./modelRoute.model');
+
+/**
+ * Used to fetch all modelRoutes data from the DB
+ * 
+ * @resolve modelRoutes data
+ */
+module.exports.getModelRoutes = () => {
+    return new Promise((resolve, reject) => {
+        ModelRouteModel
```

Comment:
you can return the thenable object and also - shouldn't ModelRouteModel be invoked here?